_dashboards/visualize/maps.md

```diff
@@ -117,12 +117,64 @@ To show a subset of the data in the index, filter the data. The following exampl
 1. Select the **Save** button.
 1. Select the **Update** button.
 
-For large datasets, you may want to avoid loading data for the whole map. To load data only for the part of the map that is currently visible, select the **Only request data around map extent** checkbox.
+For large datasets, you may want to avoid loading data for the full map. To load data only for a specific geographic area, select **Only request data around map extent**.
 {: .tip}
 
+### Drawing shapes to filter data
+
+You can filter your data by drawing shapes on your map. You can also modify the filter and remove it completely. 
+
+When drawing a shape on the map, draw it in a counterclockwise direction to ensure the most accurate datapoints are included. Drawing in a clockwise direction does not include the correct datapoints. According to the GeoJSON specification, polygon orientation must follow the right-hand rule regarding the area that it bounds.
```

Comment:
Sorry for confusion, We don't have to call out that drawing clockwise direction is incorrect.

Reply:
@VijayanB Will keep this. Thanks!